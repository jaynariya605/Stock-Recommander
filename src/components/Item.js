
const Item = ({ data, recomand }) => {
    
    return (
        <div className="list-item">
            <h3 className="list-item__title">{data.date.slice(4,)}</h3>
            <h3 className="list-item__title">${data.price}</h3>
            <h3 className="list-item__title">
                {recomand === 1 ? 'Buy' : (recomand === 0 ? 'Hold' : 'Sell')}
            </h3>
        </div>
    )
}

export default Item