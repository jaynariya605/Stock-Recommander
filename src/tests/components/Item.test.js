import Item from "../../components/Item";
import { shallow, configure } from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';

configure({
    adapter: new Adapter()
})

test('should render Item correctly', ()=>{
    const data = {
        date: new Date().toDateString(),
        price: 52,
        facebook:75
    }
    const wrapper = shallow(<Item data={data} recommand={0}/>)
    expect(wrapper).toMatchSnapshot()
})
