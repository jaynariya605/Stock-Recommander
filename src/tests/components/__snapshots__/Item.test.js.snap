// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Item correctly 1`] = `
<div
  className="list-item"
>
  <h3
    className="list-item__title"
  >
    Feb 19 2023
  </h3>
  <h3
    className="list-item__title"
  >
    $
    52
  </h3>
  <h3
    className="list-item__title"
  >
    Sell
  </h3>
</div>
`;
